analyzere/extras/visualizations.py

```diff
@@ -0,0 +1,272 @@
+import hashlib
+
+from analyzere import LayerView
+from datetime import datetime
+from graphviz import Digraph
+from IPython.display import FileLink
+from sys import float_info
+
+
+class LayerViewVisualizer(object):
+    """Class that provides simple visualization an Analyze Re LayerViews.
+
+    Using the 'graphviz' python package, this class enables users to
+    visualize Analyze Re LayerView objects.
+
+    The class has two main visualization methods:
```

Comment:
This comment is a bit redundant.  If you take a look at the output of `help(visualizations)` you'll see that this same information is displayed right below since you've included it as part of the download and view method comments.